templates/base.html

```diff
@@ -50,6 +50,7 @@
 
     <!-- Library libs -->
     <script src="static/js/typeahead.bundle.js"></script>
+    <script src="static/js/time-extend.js"></script>
```

Comment:
Do we want to keep this package? Is it useful?

Reply:
I can remove it. Was a bit of a hack anyway :)